src/main/java/org/apache/accumulo/testing/performance/tests/ScanExecutorPT.java

```diff
@@ -89,8 +95,9 @@ public Report runTest(Environment env) throws Exception {
     String tableName = "scept";
 
     Map<String,String> props = new HashMap<>();
-    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor", "se1");
-    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "heed_hints", "true");
+    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor.se1p1", "se1");
+    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor.se1p2", "se1");
+    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor.se2p1", "se2");
```

Comment:
If we just set the executor property then that will be the default for all scan types, so do not need to set it twice.  When scan_type equals dedicated, it will override it to se2.

```suggestion
    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor", "se1");
    props.put(Property.TABLE_SCAN_DISPATCHER_OPTS.getKey() + "executor.dedicated", "se2");
```

Reply:
In the test, se1 is used at two different priorities though so I thought we would need to define two scan types: se1 with priority 1 and the other se1 with priority 2. @keith-turner

Reply:
Yeah you do need two scan types.  This per table dispatcher does not care about prio, it just dispatches scans to the executor se1 or se2.  The executor se1 has further config that sets the prio based on scan type, which is where the two diff scan types matter.  So we have it dispatch everything to executor se1 unless the scan type is dedicated in which case it dispatches to se2.